xonsh/completers/path.py

```diff
@@ -151,6 +151,59 @@ def _quote_paths(paths, start, end):
     return out
 
 
+def joinpath(path):
+    if path is None:
+        return ''
+    if len(path) == 0:
```

Comment:
> But then why use the elif below?

:no_mouth:

that's a good question... i think what happened is that i wrote the bottom branches first (if/elif/else), and then found i needed the first one later and didn't bother to change the second if to elif.

i should probably be consistent....